<launch>

    <arg name="plot_raw_emg_imu" default="true" description="plot lower and upper raw EMG signals"/>

    <!-- <rosparam param="target_pos">target_0</rosparam> -->
    <node pkg="connect_myo" exec="ru_myo_node" name="RU_myo_node">
        <!-- <param name="serial_port" value="/dev/ttyACM0" />
        <param name="arm" value="RU" />
        <param name="addr" value="[18, 47, 165, 160, 46, 249]"/> -->

    </node>
    <node pkg="connect_myo" exec="rl_myo_node" name="RL_myo_node">
    </node>
    <!-- <node pkg="connect_myo" exec="connect_manager" name="Connect_manager"/> -->

    <group if="$(eval '\'$(var plot_raw_emg_imu)\' == \'true\'')">
        <node name="plot_raw_emg"
            pkg="rqt_plot"
            exec="rqt_plot"
            args="/RU_myo/imu/linear_acceleration/x 
                /RL_myo/imu/linear_acceleration/x"/>

        <!-- <node name="plot_raw_ori"
            pkg="rqt_plot"
            exec="rqt_plot"
            args="/RU_myo/imu/orientation/x
                /RU_myo/imu/orientation/y
                /RU_myo/imu/orientation/z
                /RU_myo/imu/orientation/w
            "/>
        <node name="plot_raw_ang_vel"
            pkg="rqt_plot"
            exec="rqt_plot"
            args="/RU_myo/imu/angular_velocity/x
                /RU_myo/imu/angular_velocity/y
                /RU_myo/imu/angular_velocity/z
            "/>
        <node name="plot_raw_lin_acc"
            pkg="rqt_plot"
            exec="rqt_plot"
            args="/RU_myo/imu/linear_acceleration/x
                /RU_myo/imu/linear_acceleration/y
                /RU_myo/imu/linear_acceleration/z
            "/> -->
    </group>

</launch>
